### DISCUSSION TITLE
Only allow specific packages though in baseBranch

### DISCUSSION THREAD

**Discussion by sppatel** [renovatebot/renovate#8653] [Request Help]
I have a release branch where i want renovatebot to only open PRs for dependencies with a particular sourceUrlPrefix and allowed version.  The second entry alone works fine, but also results in PR's end up being opened for additional packages that I don't want.  I added the first entry in hopes of thinking that by default the release-1.6 baseBranch would be disabled and the second packageRule entry would whitelist only the content I wanted, but by adding `enabled:false` it seems to disable the the entire branch in its entirety.  

What would be the appropriate configuration so that the release-1.6 branch only allows for PRs what I have specified in the second entry.  

Any guidance would be appreciated.

```
  "baseBranches": ["main", "release-1.6"],
  "packageRules": [
    {
      "baseBranchList": ["release-1.6"],
      "enabled": false
    },
    {
      "baseBranchList": ["release-1.6"],
      "enabled": true,
      "respectLatest": false,
      "sourceUrlPrefixes": ["https://someGitUrl"],
      "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[0-9]-foobar-1\\.6\\.[0-9]+/"
    }
  ]
```

**Comment by viceice** [comment]
which version yre you using? baseBranchList -> [matchBaseBranches](https://docs.renovatebot.com/configuration-options/#matchbasebranches), but should migraqted automatically.

https://docs.renovatebot.com/configuration-options/#matchsourceurlprefixes

**Reply by sppatel** [reply]
Yes, using an older version (aware that the configuration options have been renamed)

**Reply by sppatel** [reply]
I am attempting to change the first package rule entry to..

```
   {
      "baseBranchList": ["release-1.6"],
      "packagePatterns": "*",
      "enabled": false
    },
```
Mabye that will blacklist things correctly?

**Reply by sppatel** [reply]
Adding packagePatterns doesn't help.  So any further advice would be appreciated.

**Reply by rarkins** [reply]
Please create a reproduction repo on github.com that we can look at.

**Reply by sppatel** [reply]
will do

**Comment by sppatel** [comment]
Looks like adding in an exclusion works..

```
 "baseBranchList": ["release-1.6"],
  "packagePatterns": "*",
  "enabled": false
  "excludePackagePatterns": ["^whitelist1", "^whitelist2"],   <--- This
```
